networks

```diff
@@ -18,8 +17,10 @@ harmony
 evmos
 aurora
 boba
+nova
+
```

Comment:
I am not sure if a newline could cause trouble cause the scripts are reading the list of networks from this file. I know you wanted to separate mainnets and testnets but I would suggest to remove the newline to avoid error sources.

Reply:
fixed